paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/DefaultPaymentMethodVerticalLayoutInteractorTest.kt

```diff
@@ -79,6 +80,50 @@ class DefaultPaymentMethodVerticalLayoutInteractorTest {
         }
     }
 
+    @Test
+    fun `state displays most recently selected PM if it exists`() {
```

Comment:
Can we add a test for when we go between the different selections?

Reply:
yep! Added